.github/ISSUE_TEMPLATE/research_question.md

```diff
@@ -0,0 +1,50 @@
+---
+name: 📖 New research question
+about: Propose a new research question
+---
+
+### Context
+<!--
+  Some work is not intended to lead to new or improved code, but is
+  better considered as a research question. This doesn't fit the 
+  existing "new feature" or "bug" templates. 
```

Comment:
I don’t think you need the justification for this issue template in the issue template.